src/main/java/mops/businesslogic/FileQueryImpl.java

```diff
@@ -0,0 +1,61 @@
+package mops.businesslogic;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import mops.persistence.file.FileInfo;
+
+import java.util.List;
+import java.util.function.Function;
+
+@SuppressWarnings("PMD.LawOfDemeter") //these are a streams or database objects
+@AllArgsConstructor(access = AccessLevel.PACKAGE)
+public class FileQueryImpl implements FileQuery {
```

Comment:
can you replace the FileQuery interface with this implementation?

Reply:
sure